### ISSUE TITLE
Handle end of terminal

### ISSUE THREAD

**Issue by eliemichel** [takluyver/redsnail#11]
When the user closes the terminal, there could be a button to restart a new one.

**Comment by takluyver** [comment]
You mean if you type `exit` in the terminal? I think I'd be inclined to close the tab down, like other terminal emulators do. But having a restart button might be equally reasonable.

**Comment by eliemichel** [comment]
Yeah, either ways seem consistent, although I'd rather not have to restart the server to open a new window, nor remember the URL of the service.
